script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -76,134 +76,134 @@ The man's eyes widen as the vast expanse in front of him comes into view.
+So I'll give it everything I've got, and drag that 0% chance up to 1%.//そこに全能力を注ぎ込み、ゼロを１％に引き上げる。
 <Page36>
-In order to overcome those two worst-case scenarios, I steel myself.
-There is no consciousness left to allocate to any other event.
-So now that I've started to descend, I don't have the ability to think about how to respond to him.
+In order to overcome those two fatal scenarios, I steel myself.
+I have no mental capacity left to allocate to any other eventualities.
+So now that I've started to descend, I don't have the ability to think about how to respond to him.//俺には対応を考える機能はない。
 <Page37>
-But what if there are more than those two worst-case-scenarios?
-I do not consider that. I don't have the capacity to go beyond three branches in my mind. If more are required, victory belongs to him. That would just mean that his power is stronger than I can imagine.
+What if there are more than just two of those do-or-die moments?//もしマイナスが二ヵ所に留まらない場合は？
+I do not consider that. I don't have the capacity to overcome three 'deaths' in my mind. If that's how it plays out, victory is his. It would just mean that his power is stronger than I can imagine.
 <Page38>
-The ice spears shoot towards the falling prey like they are overlapping one after another.
+A dense flurry of ice spears shoot towards the falling prey.//氷の槍は折り重なるように、落下する獲物に殺到する。
 It's like the jaws of a beast. It's terrifying, but it doesn't matter.
-I don't know if it's because of my instincts telling me or something.
-But I was confident of what was coming. And against that frightening counterattack that should have killed me...
+I wonder if it's thanks to his instincts that he anticipated this.//そんな、ただ本能に因っただけの咀嚼など知った事か。
+Convinced that I would come for him, and terrified that I would kill him when I did...//俺がそうくると確信し、そうなれば殺されると恐れた迎撃は、
 <Page39>
 ―――Ten meters.
 The first chance for failure.
-The sensations of my limbs, the fine-tuning of my muscles, and the sudden braking of my torso and its return.
-My downward trajectory which couldn't be changed originally gets altered after witnessing that angle of fire...!
+The sensations of my limbs, the fine-tuning of my muscles, and the sudden braking of my torso return.//Sudden braking weird but I guess accurate? 手足の 感覚、筋力の 微調整、体幹の 急制動とその返し。
+After witnessing that angle of fire, I alter the natural course of my trajectory...!//本来変えられない落下軌道を、その射角を『見て』から変える……！
 <Page40>
 A lump of iron passes right next to me.
 My gamble on that '0%' chance paid off.
 One second left. Victory is mine. He is already out of options.
 <Page41>
-I've decided that the chance for failure is 'that moment'.
-I'll deal with the impact of this fall and kill him before my body shatters.
+I've decided the second critical juncture is 'that moment'.//二つ目のマイナスは『その瞬間』と決めている。
+I'll take the impact of this fall, but right before my body shatters, I'll kill him.//Not 100% on this この落下の衝撃をいなし、俺の身体が粉々になる前に、ヤツを殺す。
```

Comment:
Is this right? I'm not sure what version of いなし this is tbh but I thought this meant he would "avoid"/"parry" the shock rather than just take it
> 落下の衝撃をいなし

Reply:
It does seem like "without the impact", but it leaves me confused about what's shattering his body. I'll see if anyone else knows better.